File: flask/seed.py
from person_store import PersonStore

people = [
    {'id': 1, 'first name': 'Bart'  , 'last name': 'Simpson'},
    {'id': 2, 'first name': 'Homer' , 'last name': 'Simpson'},
    {'id': 3, 'first name': 'Marge' , 'last name': 'Simpson'},
    {'id': 4, 'first name': 'Lisa'  , 'last name': 'Simpson'},
    {'id': 5, 'first name': 'Maggy' , 'last name': 'Simpson'},
    {'id': 6, 'first name': 'Donald', 'last name': 'Duck'}   ,
    {'id': 7, 'first name': 'Daffy' , 'last name': 'Duck'}   ,
    {'id': 8, 'first name': 'Mickey', 'last name': 'Mouse'}
    ]


if __name__ == '__main__':
  ps = PersonStore()
  ps.seed(people)

File: silex/seed.php
<?php
require 'web/person_store.php';


$people = [
  [
    "first name" =>  "Bart",
    "id"         =>  1,
    "last name"  =>  "Simpson"
  ],
  [
    "first name" =>  "Homer",
    "id"         =>  2,
    "last name"  =>  "Simpson"
  ],
  [
    "first name" =>  "Marge",
    "id"         =>  3,
    "last name"  =>  "Simpson"
  ],
  [
    "first name" =>  "Lisa",
    "id"         =>  4,
    "last name"  =>  "Simpson"
  ],
  [
    "first name" =>  "Maggy",
    "id"         =>  5,
    "last name"  =>  "Simpson"
  ],
  [
    "first name" =>  "Donald",
    "id"         =>  6,
    "last name"  =>  "Duck"
  ],
  [
    "first name" =>  "Daffy",
    "id"         =>  7,
    "last name"  =>  "Duck"
  ]
];

$ps = new PersonStore();
$ps->seed($people);

?>

